### PR TITLE
Add modifiers, more special marks to cheatsheet

### LINKED ISSUE #636: The row scope does not appear in the cheatsheet
image

The above screenshot is from the current main branch. The row scope (special mark?) is not included and should be added to either the scopes or special marks section.

### LINKED ISSUE #873: Add "bounds" and "inside" to cheatsheet

### PR DESCRIPTION
**Pull request by brollin**
Closes #873, closes #636, partially addresses #667. Will check a few boxes in #840.

### CODE REVIEW

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 1]
```diff
@@ -542,6 +542,16 @@
 						}
 					]
 				},
+				{
+					"id": "excludeInterior",
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:546]: This isn't a scope type; it's a modifier. Not sure if we have a section for that?

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 2]
```diff
@@ -622,6 +632,16 @@
 						}
 					]
 				},
+				{
+					"id": "interiorOnly",
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:636]: As above, this one is also a modifier

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 3]
```diff
@@ -1073,6 +1093,36 @@
 							"description": "Nothing"
 						}
 					]
+				},
+				{
+					"id": "lineNumberModulo100",
+					"type": "mark",
+					"variations": [
+						{
+							"spokenForm": "row <#>",
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:1102]: As above I think I'd go with `<number>`?

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 4]
```diff
@@ -1073,6 +1093,36 @@
...
+							"description": "Line at line number"
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:1103]:
```suggestion
							"description": "Line number mod 100"
```

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 5]
```diff
@@ -1073,6 +1093,36 @@
...
+						}
+					]
+				},
+				{
+					"id": "lineNumberRelativeUp",
+					"type": "mark",
+					"variations": [
+						{
+							"spokenForm": "up <#>",
+							"description": "Line relative to cursor"
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:1113]:
```suggestion
							"description": "Line number relative to cursor"
```

File: cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json [hunk 6]
```diff
@@ -1073,6 +1093,36 @@
...
+						}
+					]
+				},
+				{
+					"id": "lineNumberRelativeDown",
+					"type": "mark",
+					"variations": [
+						{
+							"spokenForm": "down <#>",
+							"description": "Line relative to cursor"
```
Comment (pokey) [inline cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json:1123]:
```suggestion
							"description": "Line number relative to cursor"
```

**Review by pokey — COMMENTED** [review]
Ah sorry I really gotta add contributor docs for the new cheatsheet 😅. The defaults.json is just for making local development easier. The actual user json when the user says "cursorless cheatsheet" is generated here https://github.com/cursorless-dev/cursorless/blob/main/cursorless-talon/src/cheatsheet_html/cheat_sheet.py#L33

File: cursorless-nx/libs/cheatsheet/src/lib/cheatsheetLegend.tsx [hunk 7]
```diff
@@ -30,6 +30,11 @@ const cheatsheetLegend: CheatsheetLegend = [
     link: 'https://www.cursorless.org/docs/#targets',
     id: 'target',
   },
+  {
+    term: '#',
```
[the remark this answers is not in the document]
Comment (brollin) [reply cursorless-nx/libs/cheatsheet/src/lib/cheatsheetLegend.tsx]: Yea, I was waffling. `number` sgtm!

**Review by pokey — APPROVED** [review]
Looks good to me; made a bunch of tweaks though so lmk what you think

**Comment by brollin** [comment]
All the changes look good to me 👍

**Pull request merged**